src/test-run/index.js

```diff
@@ -475,8 +483,7 @@ export default class TestRun extends EventEmitter {
 
     // Execute command
     static _shouldAddCommandToQueue (command) {
-        return command.type !== COMMAND_TYPE.wait && command.type !== COMMAND_TYPE.setPageLoadTimeout &&
-               command.type !== COMMAND_TYPE.debug && command.type !== COMMAND_TYPE.useRole && command.type !== COMMAND_TYPE.assertion;
+        return !QUEUE_UNADDABLE_COMMANDS.includes(command.type);
```

Comment:
Let's move this check to the other [command utils](https://github.com/DevExpress/testcafe/blob/master/src/test-run/commands/utils.js)

Reply:
I think we should check that other checks work correctly for `executeExpression` command:
`isCommandRejectableByPageError`
`canSetDebuggerBreakpointBeforeCommand`